magicuidesign/magicui
# chore: optimize particles component for better performance

### What's Changed

- Added `pointer-events-none` class to the main container div in the Particles component
- This fixes the issue with carousel button interactions

### Why This Matters
The Particles component was unintentionally blocking mouse events on elements beneath it, particularly affecting carousel buttons. This simple fix allows click events to pass through to the underlying elements.

### How to test
- Run on localhost and go to `bug-recreate/particles-bug` 
- To view the bug fix make sure the `pointer-events-none` class is added in the particles.tsx file
- To view the bug make sure to remove `pointer-events-none` class from the particles.tsx file

### Next Steps
- Please review and test on your end
- Let me know if you notice any unexpected behavior
- Let me know of any other similar issue and I'll work on it :)
